format/Flight.proto

```diff
@@ -301,6 +301,36 @@ message Location {
  */
 message Ticket {
   bytes ticket = 1;
+  // Data representing some part of the data retrievable by the ticket.
+  //
+  // `inlined_completeness` indicates what part of the data retriavable
+  // by the ticket this represents. This is provided as an optimization for
+  // client/server applications that want to reduce latency to first result
+  // without requiring another RPC round-trip to retrieve the ticket.  applications
+  // built on top of Flight are responsible for any negotiation necessary on whether
+  // inlining data is appropriate.
+  //
+  // The size of inlined_data is expected to be small (typically less then 1MB) 
+  // and inlining too much data across tickets can run into underlying transport 
+  // limitations.  Furthermore, since the data is expected to be small, implementations 
+  // are less likely to optimize for zero-copy in these cases. 
+  //
+  // The inlined data is not expected to contain schema metadata. The schema
+  // should be identical to the schema provided on FlightInfo. 
+  repeated FlightData inlined_data = 2;
+  enum InlinedCompleteness {
+    // Default is no data is inlined.  An UNDEFINED value is not provided because an enum
+    // for the data that the client isn't aware of makes the data unusable anyways.
+    NO_INLINED_DATA = 0;
```

Comment:
```suggestion
    NO_DATA = 0;
```

As this is already for inline data.

Reply:
this will be changed to undefined.  I'm going to move the enum and the FlightData into a nested message

Reply:
Another idea. If the inlined data is optional, do we still need `NO_INLINED_DATA`/`undefined`?

Reply:
updating as we speak.  Will push a new version in a few minutes hopefully.  Convention for eums in protobufs is to make the first element undefined because it allows for clients to detech non-backward compatible changes by the server.

Reply:
should be updated now.